tests/org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/formatting/TypeReferenceFormatterTest.xtend

```diff
@@ -21,7 +21,7 @@ class TypeReferenceFormatterTest {
 				{
 					var �toBeFormatted� x
 				}
-			'''
+			'''.toString.trim
```

Comment:
what would be a good default behavior for the XbaseFormatter when formatting XBase expressions that are *not* embedded into another DSL? Add trailing newLine or delete all whitespace/newLines after the last token?